d2l-button-subtle.html

```diff
@@ -170,7 +170,7 @@
 			name$=[[name]]
 			type$=[[type]]>
 			<d2l-icon icon=[[icon]] class="d2l-button-subtle-icon"></d2l-icon>
-			<span class="d2l-button-subtle-content">[[text]]</span>
+			<slot><span class="d2l-button-subtle-content">[[text]]</span></slot>
```

Comment:
As discussed, I think you'll need `::slotted` here

Reply:
So it seems like `::slotted` is only needed if we were to want to style non-default content. The default seems to be fine as is. With polymer 2 `::slotted` actually only seems to work on non-default content - e.g., `button ::slotted(*)` will only apply whatever style to the non-default content, it wont actually apply to this default `span`. I tested this using the button demo pages. Let me know if there is something I might have missed.

Reply:
Oh cool -- good to know, thanks for investigating!